### ISSUE TITLE
Webapp Always On Setting

### ISSUE THREAD

**Issue by tlourey** [ansible/ansible_collections_azure#439]
##### SUMMARY
Enable 'Always On' Support in azure_rm_webapp module

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
azure_rm_webapp
azure_rm_webapp_info
##### ADDITIONAL INFORMATION

Allows the always on feature to be set via ansible. 

References: 
https://docs.microsoft.com/en-us/rest/api/appservice/webapps/createorupdateconfiguration#request-body
https://docs.microsoft.com/en-us/rest/api/appservice/webapps/getconfiguration#siteconfigresource

```yaml
- name: Create a docker web app with some app settings, with docker image
  azure_rm_webapp:
    resource_group: myResourceGroup
    name: myDockerWebapp
    plan:
      resource_group: myAppServicePlan_rg
      name: myAppServicePlan
      is_linux: true
      sku: S1
      number_of_workers: 2
    app_settings:
      testkey: testvalue
      testkey2: testvalue2
    container_settings:
      name: ansible/ansible:ubuntu1404
      alwayson: yes
```

**Comment by tlourey** [comment]
Apologies, I accidentally closed this issue. I've now reopened

**Comment by Fred-sun** [comment]
fixes by #695
